SayTheirNames/Source/View/Home/Person/PersonCell.swift

```diff
@@ -77,9 +77,10 @@ final class PersonCell: UICollectionViewCell {
     }
     
     func configure(with person: Person) {
-        profileImageView.image = UIImage(named: "man-in-red-jacket-1681010")
-        // ^Update to default place holder image when resource is added.
-        // TO-DO: Use Kilo's ImageDownloader here to DL image
+        let df = DateFormatter()
+        df.dateFormat = "dd.MM.yyyy"
+        
+        profileImageView.populate(withURL: person.images.first?.personURL ?? "")
```

Comment:
If we add the following to `Image.swift` we can remove the nil-coalescing operator here.

```
    public init(from decoder: Decoder) throws {
        let values = try decoder.container(keyedBy: CodingKeys.self)
        
        self.id = try values.decodeIfPresent(Int.self, forKey: .id) ?? -1
        self.personID = try values.decodeIfPresent(String.self, forKey: .personID) ?? ""
        self.personURL = try values.decodeIfPresent(String.self, forKey: .personURL) ?? ""
    } 
```

This way the model will always return some value for the image.

Reply:
I'm not sure I follow, the ticket originally said to add a well-mantained library for images. 

Attaching card:
https://trello.com/c/GBTondmf/37-add-an-image-loading-library

Reply:
Lets skip this for now. We can do it in another PR. Thank you.